src/bikeracks/__init__.py

```diff
@@ -42,21 +49,34 @@ def create_app(test_config=None):
         pass
         
     # register close_db and init_db_command with the app
-    from . import db
     db.init_app(app)
     
     # import and register the blueprint from the factory
-    from . import bikes
-    from . import votes
     app.register_blueprint(bikes.bikes)
     app.register_blueprint(votes.votes)
         
     @app.route('/')
     def home(name=None):
         # update map with approved markers
-        # render_template invokes the jinja template engine
         return render_template('base.html', name=name)
         
+    @app.route('/submitFeedback', methods=('POST',))
+    def submitFeedback():
+        csv_file = os.path.join(app.instance_path, "feedback.csv")
+        timestamp = int(time())
+        feedback = request.form.get('feedback', '', type=str)
```

Comment:
I think if you give a `type` argument you don't need a default value if that value is the same as just calling the type constructor.

`str() == ''` so the default value is already `''`

Reply:
removed it.